### PR TITLE
Refactoring: ThemeButton.vue

### LINKED ISSUE #7589: Refactoring: ThemeButton.vue
The CSS part should be moved into **style/** folder

### PR DESCRIPTION
**Pull request by qmonmert**
Fix #7589

### CODE REVIEW

**Comment by pascalgrimaud** [comment]
As this part is new for @qmonmert, could you have a look plz @Gnuk ?

File: src/main/style/atom/theme-button/_theme-button.scss [hunk 1]
```diff
@@ -0,0 +1,44 @@
+.jhlite-container {
```
Comment (Gnuk) [inline src/main/style/atom/theme-button/_theme-button.scss:1]: It's strange to not have names that relates to the atom name, maybe it's not well named or in the wrong place

Comment (qmonmert) [reply src/main/style/atom/theme-button/_theme-button.scss:1]: I just moved the CSS, maybe I can try to update this

**Comment by qmonmert** [comment]
@Gnuk I updated the CSS, WDYT?

**Pull request merged**